Automation/Automation/Tests/Checkout.cs

```diff
@@ -78,5 +79,46 @@ public void PlaceOrderWithoutAccount(BillingInformation billingInfo, ProductInfo
 
             //Improvement: Check the billing info and product info from Admin, not UI!
         }
+
+        public void PlaceOrderWithAccount(BillingInformation billingInfo, ProductInfo producInfo)
+        {
+            Pages.HeaderPage.GoToAccountDropdownOption(AccountOption.LOG_IN);
+            Pages.LoginPage.Login(Constants.VALID_EMAIL, Constants.VALID_PASSWORD);
+
+            Pages.HeaderPage.GoToSubcategoryFromDropdown(Category.MEN, Subcategory.Men.TEES_KNITS_AND_POLOS);
+            Pages.ProductsPage.GoToProductDetailsPage(producInfo.Name);
+
+            Pages.ProductDetailsPage.ChangeQty(producInfo.quantity);
```

Comment:
This method is really long. Add the login and adding the item to the cart steps to a [TestInitialize] method.